🧹 Turn off ETD required fields for migration

kirkkwang opened the pull request:
This commit will turn off the following required fields so we can progress with the Valkyrie migration:

:resource_type
:keyword
:degree_name
:degree_level
:degree_discipline
:degree_grantor

laritakr reviewed (commented):
Should this be based off of their current deployed version instead of main? There are some things in main that haven't yet been deployed because of the bulkrax update that didn't work right.

kirkkwang committed:
🧹 Turn off ETD required fields for migration

This commit will turn off the following required fields so we can
progress with the Valkyrie migration:

:resource_type
:keyword
:degree_name
:degree_level
:degree_discipline
:degree_grantor

kirkkwang committed:
🚧 Bring over oer_resource.yaml as is

This commit will bring over the oer_resource.yaml from Hyku as is before
any changes.  Next commit will highlight the diffs.

kirkkwang committed:
🧹 Disable resource type as a required field

This commit will temporarily disable resource type as a required field
for OER to support migration.
